### PR TITLE
chore(web): Add CheckboxField component

### PR DESCRIPTION
**Pull request by atnagata**
# Overview
Add CheckboxField component

### CODE REVIEW

File: web/src/beta/components/CheckboxField/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      {label && (
+        <Text size="xs" color={publishedTheme?.mainText || theme.main.text}>
+          {label}
+        </Text>
+      )}
+    </Field>
+  );
+};
+
+const Field = styled.div`
```
Comment (isoppp) [inline web/src/beta/components/CheckboxField/index.tsx:33]: I guess it's better to use `button` here, and it's working like `label > input`.
Then users can change the checkbox state by clicking the label as well.

File: web/src/beta/components/CheckboxField/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  height: 20px;
+  border: 1px solid ${({ theme, publishedTheme }) => publishedTheme?.weakText || theme.main.weak};
+  border-radius: 4px;
+`;
+
+const CheckMark = styled(Icon)<{ publishedTheme?: PublishTheme }>`
+  padding-left: 15%;
+  padding-right: 10%;
+  padding-top: 25%;
+  padding-bottom: 20.83%;
```
Comment (isoppp) [inline web/src/beta/components/CheckboxField/index.tsx:53]: Could you use `flex` properties in `BoxField`  to center icon?
I guess these padding properties are changing icon size.

File: web/src/beta/components/CheckboxField/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+const Field = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 12px;
+  width: 100%;
+  height: 20px;
+`;
+
+const BoxFeild = styled.button`
```
Comment (KaWaite) [inline web/src/beta/components/CheckboxField/index.tsx:38]: BoxField

File: web/src/beta/components/CheckboxField/index.tsx (hunk 2 above)
Comment (KaWaite) [reply web/src/beta/components/CheckboxField/index.tsx:50]: I think this padding directly on the Icon will also make the Icon smaller than expected

Comment (atnagata) [reply web/src/beta/components/CheckboxField/index.tsx:50]: When centered with flex properties, the design differs from FIgma.
スクリーンショット 2023-06-23 13 46 10

File: web/src/beta/components/CheckboxField/index.tsx (hunk 1 above)
Comment (atnagata) [reply web/src/beta/components/CheckboxField/index.tsx]: Fix completed.

File: web/src/beta/components/CheckboxField/index.tsx (hunk 2 above)
Comment (atnagata) [reply web/src/beta/components/CheckboxField/index.tsx]: Fix completed.

File: web/src/beta/components/CheckboxField/index.tsx (hunk 3 above)
Comment (atnagata) [reply web/src/beta/components/CheckboxField/index.tsx]: Fix completed.

File: web/src/beta/components/CheckboxField/index.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+export type Props = {
+  onClick?: (value: boolean) => void;
+  checked?: boolean;
+  label: string;
+};
+
+const CheckBoxField: React.FC<Props> = ({ onClick, checked, label }) => {
+  const theme = useTheme();
+  return (
+    <Field onClick={() => onClick?.(checked !== undefined ? checked : false)}>
```
Comment (isoppp) [inline web/src/beta/components/CheckboxField/index.tsx:14]:
```suggestion
    <Field onClick={() => onClick?.(!checked)}>
```
You have to pass opposite boolean value to the event

**Review by isoppp — APPROVED** [review]

**Review by KaWaite — APPROVED** [review]

**Pull request merged**